distribution/client/src/lib/update.js

```diff
@@ -88,6 +94,7 @@ class Update {
     });
 
     return Promise.all(tasks).then(() => {
+      this.snapshotter.snapshot(`UL{this.getCurrentLevel()}->UL${updates.meta.level} Snapshot after downloads completed, before Jenkins restart`);
```

Comment:
Please use `UI.publish` a message here about snapshotting the state, that will be exposed to the nginx status page

Reply:
done